client/src/screens/Client/Home/components/modal/NewOrEditEvent.js

```diff
@@ -182,46 +192,88 @@ const NewOrEditEvent = ({
   };
 
   const handleSaveOrUpdate = () => {
-
-    props.appointments.map((appointment) =>
-      selectedPatient.includes(appointment.patient_id)
-    );
-    if (isNewEvent) {
-      const payload = {
-        data: {
-          title: calEvent.title,
-          provider: provider,
-          patient: selectedPatient,
-          ApptStatus: calEvent.status,
-          notes: calEvent.notes,
-          start_dt: calEvent.start_dt,
-          end_dt: calEvent.end_dt
+    const submitData = () => {
+      if (isNewEvent) {
+        const payload = {
+          data: {
+            title: calEvent.title,
+            provider: provider,
+            patient: selectedPatient,
+            ApptStatus: calEvent.status,
+            notes: calEvent.notes,
+            start_dt: calEvent.start_dt,
+            end_dt: calEvent.end_dt
+          }
+        };
+        onSave(payload);
+      } else {
+        const payload = {
+          data: {
+            id: props.event.id,
+            title: calEvent.title,
+            providerName: calEvent.provider_name,
+            provider: provider,
+            patient: selectedPatient
+              ? selectedPatient
+              : {
+                id: props.event.patient_id,
+                firstname: props.event.firstname,
+                email: props.event.email
+              },
+            ApptStatus: calEvent.status,
+            notes: calEvent.notes,
+            old_start_dt: moment(props.event.start_dt).format(
+              "YYYY-MM-DD HH:mm"
+            ),
+            old_end_dt: moment(props.event.end_dt).format("YYYY-MM-DD HH:mm"),
+            new_start_dt: moment(calEvent.start_dt).format("YYYY-MM-DD HH:mm"),
+            new_end_dt: moment(calEvent.end_dt).format("YYYY-MM-DD HH:mm")
+          }
+        };
+        onEventUpdate(payload);
+      }
+    };
+    const existPatientID = props.appointments
+      .map((appointment) => selectedPatient.id === appointment.patient_id)
+      .includes(true);
+    const startTimeExist = props.appointments
+      // eslint-disable-next-line
+      .map((appointment) => calEvent.start_dt == appointment.start_dt)
```

Comment:
We added `eslint rule` but there might be some exception case and only that time we should use `eslint-disable-next-line`. 
Can you let me know the purpose of using `eslint-disable-next-line` here?

Reply:
Here i used == instead === to compare the date object but linter warned me to use === that is why i ignore the warning

Reply:
okay.